concent_api/common/tests/test_api_view.py

```diff
@@ -39,7 +40,7 @@ def setUp(self):
         self.request_factory = RequestFactory()
         self.want_to_compute = message.WantToComputeTask(
             node_name           = 1,
-            task_id             = 2,
+            task_id             = generate_uuid(),
             perf_index          = 3,
             price               = 4,
             max_resource_size   = 5,
```

Comment:
Please also adapt this area of code to ours current style

Reply:
:+1: done